tests/__init__.py

```
@@ -155,63 +138,225 @@
+}
+
+# Load specific group/category
+TEST_GROUPS = {
+	"mat": MINIMUM_ACCEPTANCE_TESTS,
+	"extra": EXTRA_TESTS,
+	"fuzzing": FUZZING_TESTS,
+	"performance": PERFORMANCE_TESTS,
+}
+
+
+def get_test_suite(group: str = None, category: str = None) -> unittest.TestSuite:
```

Ruff (0.8.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)